addons/bestja_volunteer/models.py

```diff
@@ -1,53 +1,47 @@
 # -*- coding: utf-8 -*-
+import re
+from lxml import etree
```

Comment:
LXML is external library, so, according to PEP8, it should be in the second hunk of imports.